manifests/repo.pp

```diff
@@ -32,15 +32,14 @@
 
   case $::osfamily {
     'Debian': {
-      if !defined(Class['apt']) {
-        class { 'apt': }
-      }
+      include ::apt
+      Class['apt::update'] -> Package<| |>
 
       apt::source { 'elasticsearch':
         location    => "http://packages.elastic.co/elasticsearch/${elasticsearch::repo_version}/debian",
         release     => 'stable',
         repos       => 'main',
-        key         => 'D88E42B4',
+        key         => '46095ACC8548582C1A2699A9D27D666CD88E42B4',
```

Comment:
the long key doesn't work with older puppetlabs-apt module versions which are used by older PE versions.
Once i'm able to drop those older versions i can use the longer key.